Vagrantfile

```diff
@@ -28,6 +28,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     config.vm.provision :hostmanager
   end
 
+  # Ensure role dependencies are in place
+  if ['up', 'provision'].include?(ARGV.first) && !(File.directory?('roles/Stouts.mongodb'))
```

Comment:
Is the check for role dependencies going to grow? This statement could get a bit gnarly, or there could be many of them copy/pasted. 

In what other ways might this be accomplished?